### DISCUSSION TITLE
vue-vben-admin 项目可以二次开发商用嘛？

### DISCUSSION THREAD

**Discussion by CrazyBoyM** [vbenjs/vue-vben-admin#1366] [General]
如题，原本是看到了native-admin那个项目是MIT开源协议的，正准备搭建个中后台，发现在issues里作者自己说是不能商用的要购买授 权，只能用于学习。
  我想问一下咱这个项目可以放心地用于开发自己的中后台管理项目嘛？谢谢了。

**Comment by baikefun** [comment]
demo示例中 详情见：
vben-admin 是一个基于Vue3.0、Vite、 Ant-Design-Vue 、TypeScript 的后台解决方案，目标是为中大型项目开发,提供现成的开箱解决方案及丰富的示例,原则上不会限制任何代码用于商用。
